bin/node/runtime/src/lib.rs

```diff
@@ -180,6 +181,7 @@ impl frame_system::Trait for Runtime {
 	type Event = Event;
 	type BlockHashCount = BlockHashCount;
 	type MaximumBlockWeight = MaximumBlockWeight;
+	type DbWeight = ();
```

Comment:
Is this a placeholder until we figured how much weight we want to attach to DB operations?

Reply:
yes, I think better to have no value than some random numbers I invent from thin air :)

Reply:
If we want to merge this PR before we have done full analysis of DB, then we should reconsider this obv

Reply:
Thought so. Thanks for the clarification.